src/components/Thanks/MyKiva/ThanksBadges.vue

```diff
@@ -121,6 +121,13 @@
 				/>
 			</KvExpandable>
 		</div>
+		<KvLightbox
+			:visible="showGuestAccountModal"
+			title="Finish creating your account to see what's next"
+			@lightbox-closed="showGuestAccountModal = false"
```

Comment:
Tracking events for this modal?

Reply:
There's already a ticket for all analytics for TY page: https://kiva.atlassian.net/browse/MP-1011